src/richObjectStringParser.js

```diff
@@ -33,7 +31,7 @@ export default {
 		var regex = /\{([a-z\-_0-9]+)\}/gi
 		var matches = message.match(regex)
 
-		_.each(matches, function(parameter) {
+		matches.forEach(function(parameter) {
```

Comment:
oh, you can do better. change `function (parameter)` to `parameter =>` and you can use `this` instead of `self` :wink: